### ISSUE TITLE
Very large selection / bounding box for some tree props

### ISSUE THREAD

**Issue by TuTuc0** [Bouncyrock/TaleSpire-Beta-Public-Issue-Tracker#986]
**Describe the bug**
Some trees have abnormally  large selection box, which interferes with placement and precise selection. Most trees are fine, but specially these two parts contain very large ones.

**Screenshots/Videos/Twitch Clips**
![image](https://user-images.githubusercontent.com/48619042/122055812-4d20c780-cde9-11eb-814b-d37f698228aa.png)

**Comment by baggers-br** [comment]
Agreed. We will be improving this in future. Thanks for the report!
